app/src/main/java/com/mensinator/app/settings/ExportImportDialog.kt

```diff
@@ -107,10 +103,6 @@ fun ImportDialog(
             try {
                 inputStream?.copyTo(outputStream)
                 onImportClick(file.absolutePath, selectedOption)
```

Comment:
`FileOutputStream` can throw an exception, so please move it inside the `try` block